operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/KeycloakSpec.java

```diff
@@ -67,18 +62,10 @@ public class KeycloakSpec {
     @JsonPropertyDescription("In this section you can configure Keycloak features, which should be enabled/disabled.")
     private FeatureSpec featureSpec;
 
-    public String getHostname() {
-        return hostname;
-    }
-
-    public void setHostname(String hostname) {
-        this.hostname = hostname;
-    }
-
-    @JsonIgnore
-    public boolean isHostnameDisabled() {
-        return this.hostname.equals(Constants.INSECURE_DISABLE);
-    }
+    @NotNull
```

Comment:
I think, in theory (a future dev mode or something) this can actually be `null`, right?